massa-execution-worker/src/interface_impl.rs

```diff
@@ -947,4 +998,12 @@ fn test_evm_verify() {
     // sign the message and match it with the original signature
     let (second_signature, _) = libsecp256k1::sign(&message, &private_key);
     assert_eq!(signature, second_signature);
+
+    // check 4
+    // generate the address from the public key and match it with the original address
+    // address is the last 20 bytes of the hash of the public key in raw format (64 bytes)
+    let raw_public_key = public_key.serialize();
+    let hash = sha3::Keccak256::digest(&raw_public_key[1..]).to_vec();
+    let generated_address = &hash[12..];
+    assert_eq!(generated_address, address_);
```

Comment:
Is there a reason you don't directly use the interface to prevent a desync between the tests and the interface implementation?

Reply:
Because I would either have to create an `InterfaceImpl` which requires an execution context that is way to heavy to init and not worth the hassle imo, or create another object implementing the same interface functions which would result in code duplicate anyway, even if it might be easier to match behaviours.

Reply:
Ok, it's good enough for me!